### PR TITLE
Preparing release v1.24.0

### PR DESCRIPTION
**Pull request by rubenvp8510**
Signed-off-by: Ruben Vargas <ruben.vp8510@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,6 +57,7 @@ The following table shows the compatibility of jaeger operator with different co
 
 | Jaeger Operator | Kubernetes           | Strimzi Operator   |
 |-----------------|----------------------|---------------------
+| v1.24           | v1.19+               | v0.23              |
```
Comment (jpkrohling) [inline README.md:60]: I think we should fix the versions instead of using "v1.19+". Can you list the versions explicitly?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,17 @@
...
 ==================
 
-1.23.0 (2021-07-11)
+1.24.0 (2021-07-08)
+-------------------
+* Include OIDC plugin in binary ([#1501](https://github.com/jaegertracing/jaeger-operator/pull/1501), [@esnible](https://github.com/esnible))
+* Update jaeger operator to support strimzi operator 0.23.0 ([#1495](https://github.com/jaegertracing/jaeger-operator/pull/1495), [@rubenvp8510](https://github.com/rubenvp8510))
+* Feature/add deployment strategy to crd ([#1499](https://github.com/jaegertracing/jaeger-operator/pull/1499), [@ethernoy](https://github.com/ethernoy))
+* add cassandraCreateSchema affinity ([#1475](https://github.com/jaegertracing/jaeger-operator/pull/1475), [@chasekiefer](https://github.com/chasekiefer))
+* kafka 2.5.0 is not supported by RH AMQ operator 1.7.x ([#1476](https://github.com/jaegertracing/jaeger-operator/pull/1476), [@jkandasa](https://github.com/jkandasa))
```
Comment (jpkrohling) [inline CHANGELOG.md:10]: This entry needs to be rewritten to tell what the commit brought.

**Review by pavolloffay — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (rubenvp8510) [reply CHANGELOG.md:10]: May be omitting this entry? as I think is superseded by the upgrade commit to support 0.23.

Comment (jpkrohling) [reply CHANGELOG.md:10]: Sounds good.

**Comment by rubenvp8510** [comment]
Ready for another review.

**Review by jpkrohling — APPROVED** [review]

**Pull request merged**